h2o-core/build.gradle

```diff
@@ -15,8 +16,16 @@ dependencies {
   compile "org.apache.commons:commons-math3:3.3"
   compile "commons-io:commons-io:2.4"
   compile "ai.h2o:google-analytics-java:1.1.2-H2O-CUSTOM"
-  compile "org.eclipse.jetty.aggregate:jetty-servlet:8.1.17.v20150415"
-  compile "org.eclipse.jetty:jetty-plus:8.1.17.v20150415"
+
+  compile "org.eclipse.jetty:jetty-servlet:${jetty_version}"
```

Comment:
There is another file that has jetty dependency:

./h2o-hadoop/h2o-mapreduce-generic/build.gradle:  compile "org.eclipse.jetty.aggregate:jetty-all-server:8.1.17.v20150415"